src/jdk.jlink/share/classes/jdk/tools/jlink/resources/plugins.properties

```
@@ -59,38 +59,30 @@ Class optimization: convert Class.forName calls to constant loads.
 class-for-name.usage=\
 \  --class-for-name          Class optimization: convert Class.forName calls to constant loads.
 
-compress.argument=<0|1|2>[:filter=<pattern-list>]
+compress.argument=<0|1|2|zip-[0-9]>[:filter=<pattern-list>]
```

Suggest to just show the new values.

```suggestion
compress.argument=<zip-[0-9]>[:filter=<pattern-list>]
```